mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Aida.Mixfile do
   def project do
     [
       app: :aida,
-      version: "1.1.0",
+      version: "1.2.0",
```

Comment:
This should be merged against `release/1.1.1` branch. Please rebase the PR from there.